index.bs

```diff
@@ -446,6 +451,30 @@ A {{DatagramDuplexStream}} object has the following internal slots.
 :: The getter steps are:
      1. Return [=this=].`[[Writable]]`.
 
+: <dfn for="DatagramDuplexStream" attribute>incomingMaxAge</dfn>
+:: The getter steps are:
+     1. Return [=this=]'s [=[[IncomingDatagramsExpirationDuration]]=].
+:: The setter steps are:
+     1. Set [=this=]'s [=[[IncomingDatagramsExpirationDuration]]=] to the given value.
+
+: <dfn for="DatagramDuplexStream" attribute>outgoingMaxAge</dfn>
+:: The getter steps are:
+     1. Return [=this=]'s [=[[OutgoingDatagramsExpirationDuration]]=].
+:: The setter steps are:
+     1. Set [=this=]'s [=[[OutgoingDatagramsExpirationDuration]]=] to the given value.
+
+: <dfn for="DatagramDuplexStream" attribute>incomingHighWaterMark</dfn>
+:: The getter steps are:
+     1. Return [=this=]'s [=[[IncomingDatagramsHighWaterMark]]=].
+:: The setter steps are:
+     1. Set [=this=]'s [=[[IncomingDatagramsHighWaterMark]]=] to the given value.
```

Comment:
Should we check that the value is non-negative?

Reply:
Done.